tests/frontend/test_ops_decompositions.py

```diff
@@ -285,41 +284,20 @@ def test_merge(self, hbar, tol):
         S1 = random_symplectic(n)
         S2 = random_symplectic(n)
 
-        G1 = ops.GaussianTransform(S1, hbar=hbar)
-        G1inv = ops.GaussianTransform(np.linalg.inv(S1), hbar=hbar)
-        G2 = ops.GaussianTransform(S2, hbar=hbar)
+        G1 = ops.GaussianTransform(S1)
+        G1inv = ops.GaussianTransform(np.linalg.inv(S1))
+        G2 = ops.GaussianTransform(S2)
 
         # a symplectic merged with its inverse is identity
         assert G1.merge(G1inv) is None
 
         # two merged symplectics are the same as their product
         assert np.allclose(G1.merge(G2).p[0].x, S2 @ S1, atol=tol, rtol=0)
 
-    def test_setting_hbar(self, hbar):
```

Comment:
Is hbar now determined fully through `sf.hbar`? So if a user wants to use `hbar=0.5`, they use `sf.hbar = 0.5`, and then they never have to provide it to `Gaussian`?

Reply:
Yes.

Reply:
Besides I think `GaussianTransform` was always hbar-independent since it's just a symplectic transform. `Gaussian` OTOH depends on hbar.